app/javascript/mastodon/components/status_content.jsx

```diff
@@ -64,6 +70,20 @@ class TranslateButton extends PureComponent {
 
 }
 
+const mapDispatchToProps = (dispatch, { intl }) => ({
+  openExternalLink(url) {
+    dispatch(openModal({
+      modalType: 'CONFIRM',
+      modalProps: {
+        message: intl.formatMessage(messages.openExternalLink),
+        confirm: intl.formatMessage(messages.openExternalLinkConfirm),
+        closeWhenConfirm: true,
+        onConfirm: () => window.open(url, null, 'norefferer'),
```

Comment:
Should be `noreferrer`

Reply:
Oops. You are of course absolutely right.